swift/Samples/TicTacToe/Sources/Authentication/AuthenticationWorkflow.swift

```diff
@@ -197,36 +197,51 @@ extension AuthenticationWorkflow {
+                    actions: [AlertAction(title: "Ok",
+                                          style: AlertAction.Style.default,
+                                          handler: {
+                                            sink.send(.dismissAuthenticationAlert)
+                            }
+                        )]
+                    )
+                }
+                
+            case .authorizingEmailPassword(email: let email, password: let password):
+                context.awaitResult(for: AuthorizingEmailPasswordWorker(
+                    authenticationService: authenticationService,
+                    email: email,
+                    password: password))
+                modals.append(ModalContainerScreenModal(screen: AnyScreen(LoadingScreen()), style: .fullScreen, key: "", animated: false))
 
-        case .twoFactor(intermediateSession: let intermediateSession, authenticationError: let authenticationError):
-            backStackItems.append(twoFactorScreen(
-                error: authenticationError,
-                intermediateSession: intermediateSession,
-                sink: sink))
+            case .twoFactor(intermediateSession: let intermediateSession, authenticationError: let authenticationError):
+                backStackItems.append(twoFactorScreen(
+                    error: authenticationError,
+                    intermediateSession: intermediateSession,
+                    sink: sink))
 
-        case .authorizingTwoFactor(twoFactorCode: let twoFactorCode, intermediateSession: let intermediateSession):
+            case .authorizingTwoFactor(twoFactorCode: let twoFactorCode, intermediateSession: let intermediateSession):
 
-            context.awaitResult(
-                for: AuthorizingTwoFactorWorker(
-                    authenticationService: authenticationService,
-                    intermediateToken: intermediateSession,
-                    twoFactorCode: twoFactorCode))
+                context.awaitResult(
+                    for: AuthorizingTwoFactorWorker(
+                        authenticationService: authenticationService,
+                        intermediateToken: intermediateSession,
+                        twoFactorCode: twoFactorCode))
 
-            backStackItems.append(twoFactorScreen(error: nil, intermediateSession: intermediateSession, sink: sink))
-            backStackItems.append(BackStackScreen.Item(screen: LoadingScreen(), barVisibility: .hidden))
+                backStackItems.append(twoFactorScreen(error: nil, intermediateSession: intermediateSession, sink: sink))
+                modals.append(ModalContainerScreenModal(screen: AnyScreen(LoadingScreen()), style: .fullScreen, key: "", animated: false))
         }
-
-        return ModalContainerScreen(baseScreen: BackStackScreen(items: backStackItems), modals: [])
+        return AlertContainerScreen(
+            baseScreen: ModalContainerScreen(
+                    baseScreen: BackStackScreen(
```

Comment:
Nit: The spacing is a little whack - maybe swiftformat will help later

Reply:
I also noticed some inconsistent formatting, I know @dhavalshreyas as a[ PR up for SwiftFormat ](https://github.com/square/workflow/pull/1136)and it seems to have [a no indent on case](https://github.com/square/workflow/pull/1136/files#diff-0f1374dd86acde0c7b7115e792ff72f4R38). This is something that seems to be fairly inconsistent in the codebase, specially this file.

Reply:
Probably wise to merge the Swiftformat then apply the rules to this PR...

Reply:
Yes, I found myself commenting on quite a few formatting related changes - which is a waste of everybody's time :)

Landing the `swiftformat` change shortly.

@asthatrivedi can you rebase after that lands?

Reply:
Will do!